calendars/networkx.yaml

```diff
@@ -0,0 +1,15 @@
+name: NetworkX Community Calendar
+timezone: America/Los_Angeles
+events:
+  - summary: NetworkX Community Call (Americas/Oceania)
+    description: |
+      The NetworkX Community Call. 
+
+      Meeting notes: https://hackmd.io/ea2IhUuqSrG4kM9tokXuEw
```

Comment:
FYI you can customise the link to have a user friendlier name
<img width="367" alt="Screenshot 2022-06-07 at 00 09 38" src="https://user-images.githubusercontent.com/23188539/172956424-e0a0aa13-cc90-4214-9d5b-9e8bd26cf5df.png">
.

Reply:
For some reason, I don't get that option.
![screenshot](https://user-images.githubusercontent.com/123428/172961655-d05d852d-3509-4054-a6f7-c79b08344a54.png)
So let's merge this and sort out a better name later. Maybe we need to move the hackmd to the scientific-python hackmd.

Reply:
I will follow-up on this in a future PR.

Reply:
Sure OC.

Still strange, and you are the owner of the file? Also it's strange that the address is `https://hackmd.io/@ea2IhUuqSrG4kM9tokXuEw` and not something like `https://hackmd.io/@jarrodmillman/...`.

I can do that on any files of my own, they don't need to be in the org. Maybe I clicked on some options to validate my profile or something.